docs/contributing.md

```diff
@@ -181,6 +181,16 @@ Opsdroid is an open source chatbot framework. It is designed to be extendable, s
 
  If you don't know where to start, make sure to check the [tutorials](tutorials/introduction.md) and read the [documentation](http://opsdroid.readthedocs.io/en/latest/?badge=latest).  Remember that you can also ask for help in our [matrix channel](https://riot.im/app/#/room/#opsdroid-general:matrix.org)
 
+## Creating Consistent Logging Messages
+
+When creating log messages, we would like to see a consistency around how to structure and utilize them.
+
+* In general any copyright/legal name should be Capitalized (i.e. Slack, Telegram)
+* All usage of the opsdroid capabilities should be Capitalized (Connector, Skill, Database and Parsers)
```

Comment:
I had never considered this but I like it. That makes sense to me.

Reply:
This was based on how you recently did the Telegram update.  So figured this might be the direction you wanted to go in?